src/rgw/rgw_common.h

```diff
@@ -2121,7 +2132,24 @@ struct perm_state_base {
                                                 identity(_identity),
                                                 bucket_info(_bucket_info),
                                                 perm_mask(_perm_mask),
-                                                defer_to_bucket_acls(_defer_to_bucket_acls) {}
+                                                defer_to_bucket_acls(_defer_to_bucket_acls)
+  {}
+
+  perm_state_base(CephContext *_cct,
+                  const rgw::IAM::Environment& _env,
+                  rgw::auth::Identity *_identity,
+                  const RGWBucketInfo& _bucket_info,
+                  int _perm_mask,
+                  bool _defer_to_bucket_acls,
+                  boost::optional<PublicAccessBlockConfiguration> _bucket_acess_conf) : cct(_cct),
```

Comment:
this argument should just be added to the existing constructor with a default value, ie `_bucket_access_conf = boost::none`